livre/eq/Channel.cpp

```diff
@@ -275,16 +275,9 @@ class Channel
         _renderViewPtr->setViewport( viewport );
 
         livre::Window* window = static_cast< livre::Window* >( _channel->getWindow( ));
```

Comment:
Then I suggest generateRenderingSet() is done independently of dashtreeUpdated. In the master branch, it is done twice w/o any obvious difference, hence I removed it. To me there is then no difference if the dash tree was updated or not.